index.html

```diff
@@ -67,7 +76,7 @@ <h5 class="title">Profesional Art Printing Data</h5>
                     </div>
                     <button class="button w-100">See Project</button>
                 </div>
-                <div class="card mt-24">
+                <div class="card">
                     <div class="card-works">
                         <h5 class="title">Profesional Art Printing Data</h5>
                         <p class="font-body-reg">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
```

Comment:
-[ ] 🙏Please hide the "See Project" button and only show it when hover is applied to a particular project. This is how it's shown in the original Figma design. This will help make your project look as cool🌹 as the Figma design.

Your design:
![image](https://github.com/Kccendana/portfolio_project/assets/63445083/4a0d695b-1d6a-4cd4-959a-928f2f87fcc3)

Figma design:
![image](https://github.com/Kccendana/portfolio_project/assets/63445083/c1f3e7cd-68cd-449c-b47d-34a1a3b709c2)